src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceInfo.cs

```diff
@@ -59,6 +59,15 @@ public static DiagnosticOrInterfaceInfo From(INamedTypeSymbol symbol, InterfaceD
                 }
             }
 
+            if (symbol.DeclaredAccessibility - Accessibility.Internal < 0)
```

Comment:
Same comment from below shuold be added here. I assume the ordering is important here?

Reply:
Yeah, the definition of the Accessibility enum says not to modify the order and keep it such that higher enum values are more accessible.